app/views/static_pages/faq/_faq_card.html.erb

```diff
@@ -1,17 +1,15 @@
-<div class="card faq-card">
-  <a role="button" data-toggle="collapse" data-parent="#faq-accordion" href="#faq-item-<%= index %>" aria-expanded="true" aria-controls="faq-item-<%= index %>">
+<div class="card faq-card" data-controller="reveal"> <!-- controller at top level -->
+  <a role="button" data-action="click->reveal#toggle"> <!-- toggle action to show and hide the content -->
     <div class="card-header faq-card-header" role="tab">
       <h2 class="secondary bold faq-question-title text-2xl">
         <%= faq_item[:question] %>
       </h2>
-      <i class="fas fa-plus camel expander"></i>
-      <i class="fas fa-minus camel minimizer"></i>
-    </div> <!-- /.card-header-->
+      <i class="fas fa-chevron-down"></i>
```

Comment:
Just a little polish to make the chevrons fit better with the text sizing and color on the faq item:
```suggestion
      <i class="fas fa-chevron-down text-xl text-gray-500"></i>
```